lighthouse-core/audits/autocomplete.js

```diff
@@ -199,7 +203,7 @@ class AutocompleteAudit extends Audit {
    * @param {LH.Artifacts.FormInput} input
    * @return {{hasValidTokens: boolean, isValidOrder?: boolean}}
    */
-  static checkAttributeValidity(input) {
+  static validity(input) {
```

Comment:
I think @paulirish meant to just rename the variable name?

Reply:
ohhhh I see